doc/nrf/known_issues.rst

```diff
@@ -203,6 +203,14 @@ Other issues
 
 .. rst-class:: v2-3-0
 
+NCSDK-20864: TF-M unaligned partitions when MCUboot padding and debug optimizations are enabled
```

Comment:
Looking at the doc build, I became more aware of the placement of the known issue in this file.
Should it be here, or be moved to the TF-M section?

Reply:
I think that you are right, I will move it there